applications/asset_tracker_v2/tests/ui_module/prj.conf

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_UNITY=y
+
+# Make CONFIG_EVENT_MANAGER_MAX_EVENT_CNT defined
```

Comment:
I dont think that is the only reason we include EVENT_MANAGER here. If thats the case, we can define this in CMakeLists.txt instead (like you do for `CONFIG_ASSET_TRACKER_V2_APP_VERSION_MAX_LEN`). I believe there are other reasons (like tight dependencies of the unit test on event manager itself)

I understand there are similar instances in the unit test code under asset_tracker_v2. :-)

Reply:
Yes. You're right. I tried defining it in CMakeLists.txt but got all sorts of compilation warnings. I'm not sure how to explain this properly. But I guess its somewhat obvious from the fact that it is enabled, that the test is dependent on it. I removed the comment for now but I'm happy to reinstate an alternative if you think its necessary.